guidelines.mdx

```diff
@@ -0,0 +1,519 @@
+| [Requirements](#requirements)       | Bullet points stating what the user must already have accomplished before starting any instructions on this documentation page. Necessary for How tos, Quickstarts and Tutorials. May not be relevant for other page types, e.g. FAQ, Concepts. |
+| Page body       | The content depends on the [page type](#page-type). Split long pages into subsections with [headers](#headers). Always use [numbered lists](#numbered-steps) when explaining steps a user should take. See also [page content](#page-content) |
+
+If you are creating a new page, it may be easiest to duplicate an existing page and adapt it to your requirements. See the raw files on our GitHub repo for this purpose, for example [this tutorial](https://raw.githubusercontent.com/scaleway/docs-content/main/tutorials/install-openvpn/index.mdx).
+
+More information about the different sections of each page can be found by following the links in the table above.
+
+## Page content 
+
+### Headers
+
+- There can be only one H1 on each page (the page title, defined in the [metadata](#metadata-example-and-guidance))
+- Use H2s (e.g. `## My subtitle`) to divide the page into subsections, and H3s (e.g. `### My sub-subtitle`) to divide the subsections (if appropriate). You can go up to 5 hierarchy levels (H5). In the right "Jump-to" menu, only H2s and H3s are visible and clickable.
+- Headers also serve as anchors. Link to a header anchor as follows `[text to display](#header-with-spaces-replaced-by-dashes)`
+- No headers should be orphaned. If you begin with H2, go down in sequence. The next step should either be another H2 or a level lower (in this case, H3). 
+- All headers should be in [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case). Product names should nonetheless be capitalized as appropriate.
+
+### Numbered steps
+
+When explaining to a user what to do, use **numbered steps**.
+
+- Each step should represent one action the user must do.
+- Each step should start with an imperative verb.
+- If you want to explain the effect of the action, do so in a sentence at the end of the numbered step, or on a new, non-numbered indented line.
+
+✅ **GOOD**
+
+1. Click **Instances** in the **Compute** section of the Scaleway console. The **Instances dashboard** displays.
+2. Click the Instance whose flexible IP address you want to detach. The **Overview page** for that Instance displays.
+3. Click **Detach** <Icon name="unlink" /> next to the flexible IP address you want to detach.
+
+❌ **BAD**
+
+1. Start by clicking **Instances** in the **Compute** section of the Scaleway console.
+2. You will see the **Instances dashboard**.
+3. Next, you need to find the Instance whose flexible IP address you want to detach, and click on it. You'll see the **Overview page** for the Instance. To finish the process, click **Detach** <Icon name="unlink" /> next to the flexible IP address you want to detach.
+
```

Comment:
Maybe we can put the tip under the "numbered steps" part (after the list), as I first thought the box was part of the "Bad" section